### PR TITLE
Remove css classes from Ajax Comments, when comment is unpublishev

### PR DESCRIPTION
**Pull request by slowflyer**
## Problem
After unpublish a comment while using Ajax Comments, I can not reply to existing comments below the unpublished comment, but I can still reply on comments above.

## Solution
Remove CSS classes added from Ajax comments to the reply link, when comment is unpublished.

## Issue tracker
https://www.drupal.org/project/social/issues/3222971

## How to test
*For example*
- [x] Login as CM or SM
- [x] Create topic with some comments and disable a comment in the middle.
- [x] You will see the unpublished comment.
- [x] You can still reply on comments above, but not below.

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2408-irahua3gqfyouebdhmkzawjqncbwp4cc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60e82ae957072d8d3aefe1a2

File: modules/social_features/social_comment/social_comment.module [hunk 1]
```diff
@@ -170,6 +170,9 @@ function social_comment_form_reply_validate($form, FormStateInterface $form_stat
  */
 function social_comment_comment_links_alter(array &$links, CommentInterface $entity, array &$context) {
   if (!$entity->isPublished()) {
+    if (\Drupal::moduleHandler()->moduleExists('ajax_comments')) {
```
Comment (navneet0693) [inline modules/social_features/social_comment/social_comment.module:173]: Shall we use 'social_ajax_comments' here 🙈 ?

**Review by navneet0693 — APPROVED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2408-irahua3gqfyouebdhmkzawjqncbwp4cc.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60e82ae957072d8d3aefe1a2

**Comment by slowflyer** [comment]
Any idea why Travis CI is failing?

**Comment by ronaldtebrake** [comment]
That's unrelated to your PR, thanks @slowflyer we will fix it in the main branch so we can merge those changes back in here and see tests pass.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2408-xmlcflxucw6zlppxccqb95bzmv1sxvnb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/611d11bc378c5f826b833ea6

**Comment by navneet0693** [comment]
🍒 -picked to 10.2.x, 10.1.x and 10.0.x

**Pull request merged**